hardening/host-os/ansible/main.fmf

```diff
@@ -7,25 +7,20 @@ duration: 1h
 require+:
   - openscap-scanner
 recommend+:
-  # ansible-core is not available on RHEL-7
-  # ansible is not on RHEL-8+
-  - ansible-core
-  - ansible
   # needed for the ini_file ansible plugin, and more
   - rhc-worker-playbook
 tag:
   - destructive
-  - daily
 adjust:
   - when: distro >= rhel-7 and arch == aarch64
     enabled: false
     because: RHEL-8+ requires rhc-worker-playbook which is not available for aarch64
-
-# Disabled for now because most (all?) of the failures are identical with
-# VM-using tests, /hardening/ansible, and we would prefer the waivers for
-# those tests to be categorized or bugs fixed, before we extend the test
-# matrix with this host-os ansible test.
-enabled: false
+  - when: distro == rhel-7 or distro == centos-7
+    require+:
+      - ansible
+  - when: distro >= rhel-8 or distro >= centos-8
+    require+:
+      - ansible-core
```

Comment:
Would it make sense to copy/paste this change (adjust instead of recommends) to other Ansible tests?

Reply:
I don't think it's worth it as quite soon we are going to get rid of RHEL7 completely and `ansible` requirement won't be relevant anymore